design-documents/frontend-html-binding.md

```diff
@@ -0,0 +1,18 @@
+### Rule for binding HTML values using knockout/Magento syntax
+#### Description
+Templates allow binding javascript variables/expressions using knockout/Magento to DOM elements' HTML
+like this:
+```html
+<!-- Knockout syntax -->
+<div data-bind="html: variable"></div>
+<!-- Magento syntax -->
+<div html="variable"></div>
+```
+Developers may use this binding instead of text binding for all values leaving XSS
+vulnerabilities.
+
+#### Proposing
+Create static test to require developers to use variables/functions with _'html'_ suffix
```

Comment:
How the test will recognize if `_html_` should be present or not? Or will it always fail if `data-bind:html` is present and the variable/method doesn't contain `_html_` in the name?

Reply:
always fail